packages/components/validation-message/src/ValidationMessage.tsx

```diff
@@ -0,0 +1,65 @@
+import { cx, css } from 'emotion';
+import React, { forwardRef } from 'react';
+import tokens from '@contentful/f36-tokens';
+import { Box } from '@contentful/f36-core';
+import type {
+  BoxInternalProps,
+  PolymorphicComponent,
+  PolymorphicComponentProps,
+  PolymorphicComponentWithRef,
+} from '@contentful/f36-core';
+import { ErrorCircleOutline } from '@contentful/f36-icons';
+
+const DEFAULT_TAG = 'div';
+
+const styles = {
+  icon: css({
+    marginRight: tokens.spacing2Xs,
+    marginTop: `calc(${tokens.spacing2Xs} / 2)`,
+    minHeight: '18px',
+    minWidth: '18px',
+  }),
+  root: css({
```

Comment:
I guess, it's logical to use `Flex` instead of `Box` and completely remove this rule